docs/system/system-metrics.md

```diff
@@ -33,12 +33,16 @@ instruments not explicitly defined in the specification.
-| system.cpu.utilization | Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of CPUs | 1     | Gauge                                             | Double     | state            | idle, user, system, interrupt, etc. |
-|                        |                                                                                                          |       |                                                   |            | cpu              | CPU number (0..n)                   |
+NOTE: Logical CPU refers to the virtual cores created by the operating system to manage multitasking, while a physical CPU represents the actual physical processor core(s) on the hardware.
+
+| Name                      | Description                                                                                                      | Units | Instrument Type ([*](/docs/general/metrics.md#instrument-types)) | Value Type | Attribute Key(s) | Attribute Values |
+| ------------------------- | ---------------------------------------------------------------------------------------------------------------- | ----- | ------------------------------------------------- | ---------- | ---------------- | ----------------------------------- |
+| system.cpu.time           | Seconds each logical CPU spent on each mode                                                                      | s     | Counter                                           | Double     | state            | idle, user, system, interrupt, etc. |
+|                           |                                                                                                                  |       |                                                   |            | cpu              | Logical CPU number [0..n-1]         |
+| system.cpu.utilization    | Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of logical CPUs | 1     | Gauge                                             | Double     | state            | idle, user, system, interrupt, etc. |
+|                           |                                                                                                                  |       |                                                   |            | cpu              | Logical CPU number (0..n)           |
+| system.cpu.physical.count | Reports the number of available physical CPUs - may change over time                                             | {cpu} | UpDownCounters                                    | Int64      |                  |                                     |
+| system.cpu.logical.count  | Reports the number of available logical CPUs - may change over time                                              | {cpu} | UpDownCounters                                    | Int64      |                  |                                     |
```

Comment:
I was thinking about putting it into the description instead of a note. let me know WDYT
```suggestion
| system.cpu.physical.count | Reports the number of available physical CPUs (actual physical processor cores on the hardware)                                             | {cpu} | UpDownCounters                                    | Int64      |                  |                                     |
| system.cpu.logical.count  | Reports the number of available logical CPUs (virtual processor cores created by the operating system to manage multitasking)                                              | {cpu} | UpDownCounters                                    | Int64      |                  |                                     |
```